src/library/Uno.Material/Styles/Controls/BottomNavigationBarItem.xaml

```diff
@@ -0,0 +1,152 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:controls="using:Uno.Material.Controls"
+					xmlns:converters="using:Uno.Material.Converters">
+
+	<StaticResource x:Key="BottomNavLabelFocusColorBrush"
+					ResourceKey="OnPrimaryBrush" />
+	<x:Double x:Key="BottomNavLabelFontSize">12</x:Double>
+	<StaticResource x:Key="BottomNavIconFocusColorBrush"
+					ResourceKey="OnPrimaryBrush" />
+	<StaticResource x:Key="BottomNavLabelColorBrush"
+					ResourceKey="OnPrimaryMediumBrush" />
+	<StaticResource x:Key="BottomNavIconColorBrush"
+					ResourceKey="OnPrimaryMediumBrush" />
+	<StaticResource x:Key="BottomNavNotifBackgroundColorBrush"
+					ResourceKey="ErrorBrush" />
+	<StaticResource x:Key="BottomNavNotifLabelColorBrush"
+					ResourceKey="OnErrorBrush" />
+
+	<converters:FromEmptyStringToValueConverter x:Name="EmptyToCollapsed"
+												NotNullOrEmptyValue="Visible"
+												NullOrEmptyValue="Collapsed" />
+
+	<CubicEase x:Key="EaseOutFunction"
```

Comment:
Same here, we need a longer key to avoid conflicts with resources an app could use.

Reply:
will do